packages/loader/container-loader/src/container.ts

```diff
@@ -340,7 +340,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                         // and continuously retrying (consider offline mode)
                         // Host has no container to close, so it's prudent to do it here
                         container.close(error);
```

Comment:
Add reason?

Reply:
You mean add it to error object if it supports adding properties? Or something else?

Reply:
Like so?
```suggestion
                        container.close(error, "exceptionDuringContainerLoad");
```

Reply:
Also should be `err` not `error` I think (I'm updating it to be so in my PR #6940)